### PR TITLE
[CSL-2346] remove no-ntp & NtpDrift

### PR DESCRIPTION
**Pull request by coot**
* remove `--no-ntp` from some more scripts
* `NtpDrift` should hold time difference rather than absolute value of it, added some doc strings to `NtpStatuts`.

### CODE REVIEW

File: networking/src/Ntp/Client.hs [hunk 1]
```diff
@@ -42,10 +42,13 @@ import           Ntp.Util (createAndBindSock, resolveNtpHost, selectIPv4, select
                            udpLocalAddresses, withSocketsDoLifted)
 
 data NtpStatus =
+      -- | The difference between ntp time and local system time
       NtpDrift Microsecond
```
Comment (avieth) [inline networking/src/Ntp/Client.hs]: Such that if you add it to the local time, then you get the NTP time?

File: networking/src/Ntp/Client.hs [hunk 2]
```diff
@@ -124,8 +127,7 @@ handleCollectedResponses cli = do
     handler (newMargin, transmitTime) = do
         let ntpTime = transmitTime + newMargin
         localTime <- realTime
-        let timeDiff = abs $ ntpTime - localTime
-        atomically $ writeTVar (ncStatus cli) (NtpDrift timeDiff)
+        atomically $ writeTVar (ncStatus cli) (NtpDrift $ ntpTime - localTime)
```
Comment (avieth) [inline networking/src/Ntp/Client.hs]: Right-o

File: networking/src/Ntp/Client.hs (hunk 1 above)
Comment (coot) [reply networking/src/Ntp/Client.hs]: Yes

**Comment by coot** [comment]
@avieth the appveyor build fails with a 404 error when trying to download `https://slproweb.com/download/Win64OpenSSL-1_0_2n.exe` which is unavailable (there is a newer version available instead). I asked on `sl-devops` but nobody responded.

**Comment by coot** [comment]
The necessary change was just merged; I rebased, so everything should be ok.

**Comment by coot** [comment]
`ag "no-ntp"` returns no results.

**Pull request merged**